# creating custom exception
class NegativeNumberError(Exception):
  pass
def check_positive(number):
  if number < 0:
    raise NegativeNumberError("Negative number entered!")
try:
  num = int(input("Enter a positive number:"))
  check_positive(num)
  print("Number you entered is positive number")
except NegativeNumberError as e:
  print(e)

